db/seeds.rb

```diff
@@ -1,5 +1,32 @@
 locations = {
   meetups: [
+    {
+      slug: 'ruby-tuesday-tallinn',
+      name: 'Ruby Tuesday Tallinn',
+      description: 'Ruby User Group meeting every last Tuesday of the month from 18:00 onward.',
+      latitude: 59.398279,
+      longitude: 24.659693,
+      link: 'https://www.facebook.com/events/785659903522120',
+      img: './ruby-tuesday-tallinn.jpg'
+    },
+    {
+      slug: 'ruby-belgium',
+      name: 'Ruby Belgium',
+      description: 'Ruby Belgium is a community of programming enthusiasts using ruby as a tool to connect, build and share knowledge',
+      latitude: 50.84792799176271,
+      longitude: 4.3534307855468235,
+      link: 'https://rubybelgium.be/',
+      img: 'https://rubybelgium.be/assets/images/ruby-ecc3a010.svg'
+    },
+    {
+      slug: 'ruby-banitsa',
+      name: 'Ruby Banitsa',
+      description: 'We get together in Sofia, Bulgaria, nom some fresh pastry and talk about tech. All this in a casual setting.',
+      latitude: 42.66169767743095,
+      longitude: 23.317954446819098,
+      link: 'https://rubybanitsa.com',
+      img: './ruby-banitsa.svg'
```

Comment:
the same as in line 10, this should link to public/logos - since you've put the logos there (as we asked for, so thank you!). So it should be `/logos/ruby-banitsa.svg` - just like in https://github.com/rubyeurope/rubyeurope.com/pull/1/files#diff-c61a903a1201b0c7f076b0270377c20390c26f655ce63689d7d843296fcf63f3R287